src/error.rs

```diff
@@ -8,7 +8,7 @@ pub enum Error {
     InvalidDigit(u8),
 
     /// Value out of range.
-    OutOfRange(i32),
+    OutOfRange(u16),
```

Comment:
As a type for containing out of range values, `u16` kind of sucks because its range is very similar to the range of allowable values. Better if this were allowed to contain large positive and negative numbers.

Reply:
Firstly, signed integers do not make sense here because `RomanUnitIterator` doesn't yield negative integers.

`u16` gives us `(2^16) - 4999 = 60537` out of range values. In my opinion, that's enough, but if you don't think so, I can change it to `u32` (`4294962297` out of range values).

Reply:
Well, for some reason, I was thinking the constructor still took an `i32`, but that has also been changed. I think you've sold me.